tests/NLog.UnitTests/Targets/DatabaseTargetTests.cs

```diff
@@ -759,6 +761,120 @@ public void OdbcShorthandNotationTest()
             Assert.Equal(typeof(System.Data.Odbc.OdbcConnection), dt.ConnectionType);
         }
 
+        [Fact]
+        public void Oracle_SetParamTypeShouldSetCustomEnumValueTest()
+        {
+            var expected = OracleDbType.Clob;
+            var target = new DatabaseTarget
+            {
+                Name = "notimportant",
+                DBProvider = "notimportant",
+                ConnectionString = "notimportant",
+                CommandText = "notimportant",
+            };
+            var parameterInfo = new DatabaseParameterInfo
+            {
+                DbType = typeof(OracleDbType).FullName + ".Clob"
+            };
+            var parameter = new MockDbParameter(new MockDbCommand(), 0); 
+
+            target.SetParamType(parameter, parameterInfo.DbType);
+
+            Assert.Equal(expected, parameter.OracleDbType);
+        }
+
+        [Fact]
+        public void DbType_SetParamTypeShouldSetEnumValueTest()
+        {
+            var expected = DbType.DateTime2;
+            var target = new DatabaseTarget
+            {
+                Name = "notimportant",
+                DBProvider = "notimportant",
+                ConnectionString = "notimportant",
+                CommandText = "notimportant",
+            };
+            var parameterInfo = new DatabaseParameterInfo
+            {
+                DbType = typeof(DbType).FullName + "." + expected,
```

Comment:
I prefer to write the full string out, so we the unit test is kinda docs. Agree?